src/directLine.ts

```diff
@@ -333,12 +334,16 @@ export class DirectLine implements IBotConnection {
             if (options.token && options.conversationId) {
                 this.streamUrl = options.streamUrl;
             } else {
-                console.warn('streamUrl was ignored: you need to provide a token and a conversationid');
+                console.warn('DirectLineJS: streamUrl was ignored: you need to provide a token and a conversationid');
             }
         }
 
-        if (options.pollingInterval !== undefined) {
-            this.pollingInterval = options.pollingInterval;
+        if (typeof options.pollingInterval === 'number') {
```

Comment:
`typeof NaN === 'number'` <- true 
This should change to guard agains NaN. e.g.
```js
const interval = Math.max(~~options.pollingInterval, POLLING_INTERVAL_LOWER_BOUND);
```
Because `~~NaN === 0`

Reply:
That's a great classical 140byt.es tips. :)

Reply:
good call